azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -678,6 +678,24 @@ def stop_tracking_unit_cgroups(self, unit_name):
             except Exception as exception:
                 logger.info("Failed to stop tracking resource usage for the extension service: {0}", ustr(exception))
 
+        def stop_tracking_extension_cgroups(self, extension_name):
+            """
+            TODO: Memory tracking
+            """
+            try:
+                extension_slice_name = SystemdCgroupsApi.get_extension_cgroup_name(extension_name) + ".slice"
+                cgroup_relative_path = os.path.join('azure.slice/azure-vmextensions.slice',
+                                                    extension_slice_name + ".slice")
```

Comment:
There seems to be a typo, an extra `.slice` here - 
`SystemdCgroupsApi.get_extension_cgroup_name(extension_name) + ".slice" + ".slice"`

If this is used in multiple other places, maybe make this a function too (something like `SystemdCgroupsApi.get_extension_slice_name(extension_name)`) or something like that